File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(S):
    answer = True
    st = ''
    for s in S:
        st+=s.lower()
    p = st.count('p')
    y = st.count('y')
    
    if p == 0 and y == 0:
        pass
    else:
        if p!=y:
            answer = False

    return answer

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    let ss = s.toUpperCase()
    let pCnt = 0
    let yCnt = 0
    for (let s_ of ss){
        if (s_ === 'P'){
            pCnt += 1
        } else if(s_==='Y'){
            yCnt += 1
        }
    }
    if (pCnt !== yCnt){
        answer = false 
    }
    

    return answer;
}